### ISSUE TITLE
Check Lost Samples num when read from perf event buffer encounter EOF

### ISSUE THREAD

**Issue by hengyoush** [hengyoush/kyanos#225]
**Is your feature request related to a problem? Please describe.**
Check Lost Samples num when read from perf event buffer encounter EOF, currently only print a error log like `[dataReader] handle syscall data err: EOF`.  

We need to **log it properly** and consider to save a `lost sample count` statistic count for displaying to the tui latter (Displaying to the tui might be another task).

**Describe the solution you'd like**
in bpf/events.go

```go
func PullXXXDataEvents(..) {
 ...

	if record.LostSamples > 0 {
		// update counter here
	}
 ...
}
```

**Comment by spencercjh** [comment]
So, as a first step for this story, I add a `if record.LostSamples>0` after `record, err := reader.Read()` for each `PullXXXEvent`, introducing some `atomic.Uint64` counters. Wouldn't that be ready to show them in the TUI?

**Comment by hengyoush** [comment]
Yes, it is enough. But i haven't decided how to present it to TUI properly yet. Feel free to share any suggestions.
